pkg/prm/tool.go

```diff
@@ -1,73 +1,84 @@
-//nolint:structcheck,unused
+// nolint:structcheck,unused
 package prm
 
 import (
 	"io"
-
-	"github.com/Masterminds/semver"
 )
 
 type Tool struct {
-	stdout   io.Reader
-	stderr   io.Reader
-	exitCode ToolExitCode
-	cfg      ToolConfig
+	Stdout   io.Reader
+	Stderr   io.Reader
+	ExitCode ToolExitCode
+	Cfg      ToolConfig
 }
 
 type ToolExitCode int64
 
 const (
-	SUCCESS ToolExitCode = iota
-	FAILURE
+	FAILURE ToolExitCode = iota
+	SUCCESS
```

Comment:
Doesn't this flip failure to be a `0` and success a `1`?

Reply:
Yes, it does. And from a go point of view it means the default value is a failure.

Reply:
Okay, that scans!